docs/en/reference/introduction.rst

```diff
@@ -40,11 +40,11 @@ PHAR
 To install by downloading the PHAR, you just need to download the latest PHAR file from the
 releases_ page on GitHub.
 
-Here is an example using the ``1.7.2`` release:
+Here is an example using the ``2.0.0`` release:
 
 .. code-block:: sh
 
-    wget https://github.com/doctrine/migrations/releases/download/v1.7.2/doctrine-migrations.phar
+    wget https://github.com/doctrine/migrations/releases/download/v2.0.0/doctrine-migrations.phar
```

Comment:
I think that for those users that just copy the command from the documentation it would be more convenient to get the current version